src/wayland/compositor/mod.rs

```diff
@@ -105,6 +105,19 @@ struct Marker<R> {
     _r: ::std::marker::PhantomData<R>,
 }
 
+/// New buffer assignation for a surface
+pub enum BufferAssignation {
```

Comment:
Apparently "assignation" really is a word, but I would expect "assignment". English is not my first language, so I'm not entirely sure.

Reply:
Sounds weird for sure, but might be a *german* bias. :laughing:

Reply:
Ah well, I guess it's possibly a frenchism from me. :sweat_smile: